### PR TITLE
PS-7367 Update OpenSSL versions in Installation (5.6)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified source/installation.rst

### CODE REVIEW

File: doc/source/installation.rst [hunk 1]
```diff
@@ -39,10 +39,25 @@ Installing |Percona Server| from a Binary Tarball
...
+   :header-rows: 1
+
+   * - OpenSSL version
+     - Linux Distrubutions
+   * - ssl098
+     - Only for *Debian Squeeze* (``libssl.so.0.9.8 => /usr/lib/libssl.so.0.9.8 (0x00007f9b30db6000)``);
+   * - ssl098e
+     - Only for *CentOS* 5 (``libssl.so.6 => /lib64/libssl.so.6 (0x00002aed5b64d000)``);
+   * - ssl100
+     - For all *Debian/Ubuntu* versions except *Debian Squeeze* (``libssl.so.1.0.0 => /usr/lib/x86_64-linux-gnu/libssl.so.1.0.0 (0x00007f2e389a5000)``);
```
Comment (hrvojem) [inline doc/source/installation.rst]: these distributions aren't supported anymore

File: doc/source/installation.rst [hunk 2]
```diff
@@ -39,10 +39,25 @@ Installing |Percona Server| from a Binary Tarball
...
+   * - ssl101
+     - For *CentOS* 6 and *CentOS* 7 (``libssl.so.10 => /usr/lib64/libssl.so.10 (0x00007facbe8c4000)``);
+   * - ssl102
+     - For *Debian* 9 and *Ubuntu* versions starting from 14.04 (``libssl.so.1.1 => /usr/lib/libssl.so.1.1``);
```
Comment (hrvojem) [inline doc/source/installation.rst:52]: Ubuntu 14.04 isn't supported anymore

**Review by hrvojem — CHANGES_REQUESTED** [review]

File: doc/source/installation.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/installation.rst:52]: Removed platforms which has reached EOL.

**Review by hrvojem — APPROVED** [review]
LGTM

**Pull request merged**